classes/02class/exercises/c02-network01/lgothelipe/c02-network01.txt

```diff
@@ -0,0 +1,34 @@
+10.0.0.1/24
+Ip binary = 00001010.00000000.00000000.00000001
+Subnet binary = 11111111.11111111.11111111.00000000
+Subnet decimals = 255.255.255.0
+private/public = private
+hosts = 251
+
+192.168.0.1/16
+Ip binary = 10100000.10101000.00000000.00000001
+Subnet binary = 11111111.11111111.00000000.00000000
+Subnet decimals = 255.255.0.0
+private/public = private
+hosts = 65,531
+
+249.165.166.135/30
+Ip binary = 11111001.10100101.10100110.10000111
+Subnet binary = 11111111.11111111.11111111.11110000
+Subnet decimals = 255.255.255.240
+private/public = public
+hosts = 11
+
+236.68.223.18/32
+Ip binary = 11111001.10100101.10100110.10000111
+Subnet binary = 11111111.11111111.11111111.11111100
+Subnet decimals = 255.255.255.252
+private/public = public
+hosts = 0
```

Comment:
that would be 1 host, as we have 2^0 = 1. But in AWS, the minimum subnet size is /28, so a /30 subnet is not valid in AWS.